code_samples/front/add_design/config/packages/views.yaml

```diff
@@ -1,6 +1,6 @@
 ibexa_design_engine:
     design_list:
-        summersale: [summersale, standard]
+        summersale: [summersale]
```

Comment:
If you remove standard here, the following doc in https://doc.ibexa.co/en/master/templating/design_engine/add_new_design/#add-themes stops making sense, as we refer to it later.

Reply:
@DominikaK 
This could be the occasion to remind than [the `standard` theme is always added to the end of each design's theme list](https://github.com/ezsystems/ezplatform-standard-design/blob/0.3/src/bundle/DependencyInjection/Compiler/StandardThemePass.php#L36) (if not already in it). [The `standard` design is always added to the design list with an empty theme list](https://github.com/ezsystems/ezplatform-design-engine/blob/3.0/bundle/DependencyInjection/EzPlatformDesignEngineExtension.php#L42) and the `standard` theme is added to it.

Reply:
That's a good one, I wasn't even aware of it. Let's add an explanation here.